controllers/namespacescope_controller.go

```diff
@@ -305,7 +305,13 @@ func (r *NamespaceScopeReconciler) DeleteAllRbac(instance *operatorv1.NamespaceS
 		return err
 	}
 
-	for _, toNs := range instance.Spec.NamespaceMembers {
+	usingMembers, err := r.getNamespaceList(instance)
+	if err != nil {
+		return err
+	}
+	deletedMembers := util.GetListDifference(instance.Spec.NamespaceMembers, usingMembers)
```

Comment:
we should not remove the user added namespaces' rbac in `common-service` CR.

Reply:
We don't remove rbac that user added into common-service CR, this case will be triggered only when common-service CR is deleted.